specs/cli/plugin-add.md

```diff
@@ -16,17 +16,40 @@ Developers can generate `openai` and `apimanifest` type of plugins. By generatin
 
 Once the `workspace.json` file is generated and the OpenAPI description file is saved locally, the generation will be executed and the plugin and the sliced OpenAPI description will become available.
 
+For `openai` plugins, the mapping should follow [Hidi logic to generate OpenAI Plugin](https://github.com/microsoft/OpenAPI.NET/blob/vnext/src/Microsoft.OpenApi.Hidi/OpenApiService.cs#L748). Requiring fields default as the following:
+
+| OpenAI field | Default value |
```

Comment:
How can we customize these fields? It isn't clear for me and I feel we should have arguments on the CLI to provide these values?

Reply:
I have update the info based on our conversation yesterday. Do you still think we need to incorporate changes? In case the OpenAPI document doesn't have extensions we will default to basic information like Title and Description.